README.md

```diff
@@ -46,7 +45,85 @@ Take notice that this project is build in a way that gives the developer a lot o
 1. `Watch` this project to get notified about new releases, issues, etc.
 1. Update Go, Mage and GolangCI-Lint version in [Dockerfile](Dockerfile). Take notice that when working on bare metal or via SSH, then you should also to do it manually on your machine.
 1. Configure linters via [.golangci.yml](.golangci.yml).
-1. Develop tasks/targets in [magefile.go](magefile.go) and [.vscode/tasks.json](.vscode/tasks.json).
+1. Develop Mage targets in [magefile.go](magefile.go) and assosiated tasks in [.vscode/tasks.json](.vscode/tasks.json).
+
+Notable files:
+
+- [.devcontainer](.devcontainer/devcontainer.json) - Visual Studio Code Remote Cotainer configs
+- [.github](.github/workflows/build.yml) - GitHub Action workflow (CI build)
+- [.vscode](.vscode) - Visual Studio Code configs
+- [.golangci.yml](.golangci.yml) - GolangCI-Lint config
+- [docker-compose.yml](docker-compose.yml) - Compose file used in [CI Build](.github/workflows/build.yml)
+- [Dockerfile](Dockerfile) - Builder image used in [docker-compose.yml](docker-compose.yml) and [.devcontainer](.devcontainer/devcontainer.json)
+- [magefile.go](magefile.go) - Mage targets used in [docker-compose.yml](docker-compose.yml) and [.vscode/tasks.json](.vscode/tasks.json)
+
+## FAQ
+
+### Why Mage instead of Make
+
+Here is [why](https://github.com/magefile/mage#why).
+However, updating to Make is pretty straightforward.
+
+1. Replace [magefile.go](magefile.go) with a `Makefile` file:
+
+```make
+.DEFAULT_GOAL := help
+
+.PHONY: all
+all: ## full build: build, lint, test
+all: build lint test
+
+.PHONY: build
+build: ## go build
+	go build ./...
+
+.PHONY: lint
+lint: ## golangci-lint
+	golangci-lint run
+
+.PHONY: test
+test: ## go test with race detector and code covarage
+	go test -race -covermode=atomic
+
+.PHONY: help
+help:
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+```
+
+1. Remove Mage installation from [Dockerfile](Dockerfile).
+1. Update [docker-compose.yml](docker-compose.yml) and [.vscode/tasks.json](.vscode/tasks.json) to use `make`.
+
+If you want to use Make on bare-metal Windows, then you can:
+
+1. use [WSL (Windows Subsystem for Linux)](https://docs.microsoft.com/en-us/windows/wsl/install-win10) and [Visual Studio Code Remote SSH extension](https://code.visualstudio.com/docs/remote/wsl),
```

Comment:
```suggestion
1. Use [WSL (Windows Subsystem for Linux)](https://docs.microsoft.com/en-us/windows/wsl/install-win10) and [Visual Studio Code Remote SSH extension](https://code.visualstudio.com/docs/remote/wsl),
```

Half of the points start with lowercase and the second half with uppercase.

Reply:
Changed. Please review